freelist_test.go

```diff
@@ -142,12 +143,12 @@ func TestFreelist_releaseRange(t *testing.T) {
 				{id: 9, n: 2, allocTxn: 175, freeTxn: 200},
 			},
 			releaseRanges: []testRange{{50, 149}, {151, 300}},
-			wantFree:      []pgid{4, 9},
+			wantFree:      []pgid{4, 9, 10},
```

Comment:
why this is changed?

Reply:
because the old test is wrong

Reply:
because the old test is wrong, the page 9 and 10 can be both freed, because it is in release range {151, 300}.

Reply:
why the old test can be passed then?

Reply:
because it doesn't free overflow page(page10 )

Reply:
it shouldn't test like that IMHO.

Reply:
can you fix this test in a separate PR? thanks!

Reply:
will do